scripts/instances/deploy-dhis2-db.sh

```diff
@@ -12,6 +12,8 @@ NAME=$2
 DATABASE_ID=${DATABASE_ID:-1}
 DATABASE_SIZE=${DATABASE_SIZE:-5Gi}
 INSTANCE_TTL=${INSTANCE_TTL:-""}
+DB_RESOURCES_REQUESTS_CPU=${DB_RESOURCES_REQUESTS_CPU:-250m}
```

Comment:
Same with DB as mentioned above, we're already in that context. No need to add the prefix